controlsfx/src/main/java/org/controlsfx/property/editor/NumericField.java

```diff
@@ -77,6 +77,10 @@ public NumericField( Class<? extends Number> cls ) {
         }
     }
 
+    protected void setValue(Number v) {

```

Comment:
If we do keep this method (see @eugener comment), it will need a javadoc.

Reply:
Fixed